### ISSUE TITLE
Replace curly braces for arrays in format.php

### ISSUE THREAD

**Issue by toza-mimoza** [jmvedrine/moodle-qformat_qtitwo#3]
Hi!

Can you please add me as a collaborator so I can commit a change to allow this plugin to work on recent Moodle? 

To fix the following error: "Array and string offset access syntax with curly braces is no longer supported", curly braces need to be replaced with "[ ]" to conform to the PHP 7.4 standard. After that, the plugin works as intended and exports questions in the QTI 2.0 format.

**Comment by jmvedrine** [comment]
Hello,
I have invited you to collaborate to this repo.
But as I am now old and I stopped all Moodle programming (and in fact nearly all programming activities), I would prefer that you fork this repo and that you become the new maintainer of this plugin. If you agree, I can explain how to do that and tell the plugin library responsable at Moodle organisation to agree you as the new maintainer of this plugin. Thanks.
